muffnn/mlp/base.py

```diff
@@ -284,7 +284,11 @@ def _set_up_graph(self):
                            scope='layer_%d' % i, sparse_input=True)
             else:
                 if self.keep_prob != 1.0:
-                    t = tf.nn.dropout(t, keep_prob=self._keep_prob)
+                    if self.activation is tf.nn.selu:
+                        t = tf.contrib.nn.alpha_dropout(
```

Comment:
it seems like it'd be better to have a `dropout_type` kwarg control this, unless it's the case that that alpha dropout could never be used except with selu activations and vice versa.

Reply:
So alpha dropout is only meant for selu activations. The technique was invented in the paper on SELU and is designed explicitly for that. Thus I think this hard coding is better.

Reply:
ah ok.  could you add a note to one of the docstrings?  i'm not sure whether the class docstring or method kwarg documentation is more appropriate.  probably the kwarg (e.g., "Setting this to `tf.nn.selu` will also cause alpha dropout to be used, implementing a Self-Normalizing Neural Network (Klambauer et al., 2017).").